_data/retreat_presenters.yml

```diff
@@ -0,0 +1,17 @@
+- name: Eden Thomson
+  image: assets/images/presenters/eden-thomson.png
```

Comment:
RE: image handling

This should change to `image: eden-thomson.png`

And you can put the image in `img/`.